src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -118,7 +118,8 @@
      (format "sweep of stale reports (threshold: %s)"
              (format-period report-ttl))
      (jdbc/with-transacted-connection db
-       (scf-store/delete-reports-older-than! (ago report-ttl))))
+       (scf-store/delete-reports-older-than! (ago report-ttl))
+       (scf-store/delete-old-unassociated-catalogs! (ago report-ttl))))
```

Comment:
it feels like this function should be renamed from sweep-reports!, since it deals with catalogs too. Would it make any sense to have a catalog-ttl for setting the expiration independent of reports?